index.html

```diff
@@ -2169,7 +2176,7 @@ <h3>The <dfn>NFCScanOptions</dfn> dictionary</h3>
         title="Filter accepting only JSON content from https://www.w3.org"
         class="example highlight">
         const options = {
-          url: "https://www.w3.org/*",  // any path from the domain is accepted
+          id: "https://www.w3.org/*",  // any path from the domain is accepted
```

Comment:
I am thinking we should continue using `url` in scan options, or otherwise we should make clear it's a pattern the `id` is tested against, but calling it `id` gives the false assumption that a full id is matched.

Reply:
- if you add an url, it will be an exact match
- only if you add * and the like it is more flexible - that is the same for mediaType currently.
- I don't see how url is any clearer than id (documentation will be needed anyway)

We could consider calling it idFilter and mediaTypeFilter - but that should be another GH issue

Reply:
or maybe `containsId` and `containsMediaType` - that seems to match the behavior

Reply:
Filed an issue for this